### ISSUE TITLE
Modals on results page don't return focus to triggering button on close

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4081]
## Area or Component
Result page - actions menu for each result
Also on the person page with the "archive person" modal

## Quick Summary
The keyboard focus doesn't return to the triggering button when a modal is closed for most of the actions on the result page (print result, text result, etc)

## Accessibility Violation
Modal is closed, focus is not returned to trigger: https://cdc-devtools.dequecloud.com/rule-help/en/focus-modal-not-returned

## Action Requested
Return keyboard focus to the triggering element instead of to the `<body>`. 

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1978534823&range=7:7

**Comment by BobanL** [comment]
Is it enough to return focus to the actions button (the three dots) 
![image](https://user-images.githubusercontent.com/10108172/192885601-2eda4438-6d19-4911-816a-7ba851e1f97b.png)

 or should the focus be returned to the specific button (like print results)? 
![image](https://user-images.githubusercontent.com/10108172/192885571-3fe1b86f-7d9b-4306-9655-29101d65c1bd.png)

**Comment by emmastephenson** [comment]
@BobanL ideally we'd return focus to the specific button, like print results.
